make/test.mk

```diff
@@ -43,7 +43,7 @@ test-e2e-host-local:
 e2e-run:
 	oc get kubefedcluster -n $(HOST_NS)
 	oc get kubefedcluster -n $(MEMBER_NS)
-	oc new-project $(TEST_NS) --display-name e2e-tests 1>/dev/null
+	-oc new-project $(TEST_NS) --display-name e2e-tests 1>/dev/null
```

Comment:
```suggestion
	oc new-project $(TEST_NS) --display-name e2e-tests 1>/dev/null
```

Reply:
`-oc` is to ignore error in case namespace already created previously.  This is useful as for host_operator and reg_service we are using same namespace `$(HOST_NS)`

Reply:
Discussing here https://github.com/codeready-toolchain/toolchain-e2e/pull/12#discussion_r336439458